problems/3.conditionals.js

```diff
@@ -1,26 +1,42 @@
-/** Returns "It's freezing!" if the temperature is below 32, and "It's alright" otherwise
+/** Returns "It's freezing!" if the temperature is below 32, 
+ * and "It's alright" otherwise
 * @param {number} temp - The current temperature
 * @returns {string} - A message about the temperature
 */
-function temperatureMessage() {
-
+const temperatureMessage = (temp) => {
+  if (temp <= 32) {
+    return "It's freezing!"
+  } else {
+    return "It's alright"
+  }
 }
 
-/** Returns "It's freezing!" if the temperature is below 32, and "It's alright" otherwise. The implementation must use a ternary.
+/** Returns "It's freezing!" if the temperature 
+ * is below 32, and "It's alright" otherwise. 
+ * The implementation must use a ternary.
 * @param {number} temp - The current temperatur
 * @returns {string} - A message about the temperature
 */
-function temperatureMessageUsingTernary() {
-
+const temperatureMessageUsingTernary = (temp) => {
+  return (temp <= 32 ? "It's freezing!" : "It's alright");
 }
 
 
-/** Returns "You are a senior" if the age is 65 or older, "You are an adult" if the age is 18 or older, and "You are a minor" otherwise.
+/** Returns "You are a senior" if the age is 65 or older, 
+ * "You are an adult" if the age is 18 or older, 
+ * and "You are a minor" otherwise.
 * @param {number} age - The age of the user
 * @returns {string} - A message about their age
 */
-function ageMessage() {
-
+const ageMessage = (age) => {
+  if (age >= 65) {
+    return "You are a senior"
+} 
+else if (age >= 18 && age <= 65) {
```

Comment:
Looks good. No need to check the end. If we made it past the initial if we already know it's less than 65. 

Also use a code formatter like prettier.